custom_components/enphase_envoy_custom/envoy_reader.py

```diff
@@ -217,28 +223,30 @@ async def _fetch_owner_token_json(self) :
         async with self.async_client as client:
             # login to the enlighten UI
 
-            resp = await client.get(ENLIGHTEN_AUTH_FORM_URL)
-            soup = BeautifulSoup(resp.text, features="html.parser")
+            #resp = await client.get(ENLIGHTEN_AUTH_FORM_URL)
+            #soup = BeautifulSoup(resp.text, features="html.parser")
             # grab the single use auth token for this form
-            authenticity_token = soup.find('input', {'name': 'authenticity_token'})["value"]
+            #authenticity_token = soup.find('input', {'name': 'authenticity_token'})["value"]
             # and the form action itself
-            form_action = soup.find('input', {'name': 'authenticity_token'}).parent["action"]
-            payload_login = {
-                'authenticity_token': authenticity_token,
-                'user[email]': self.enlighten_user,
-                'user[password]': self.enlighten_pass,
-            }
-            resp = await client.post(ENLIGHTEN_AUTH_FORM_URL+form_action, data=payload_login, timeout=10)
+            #form_action = soup.find('input', {'name': 'authenticity_token'}).parent["action"]
+
+            # Get an enlighten user session
+            payload_login = {'user[email]': self.enlighten_user,'user[password]': self.enlighten_pass}
+
+            resp = await client.post(ENLIGHTEN_AUTH_JSON_URL, data=payload_login)
```

Comment:
Can you talk about why you've commented out this code and the use of the authenticity_token? I'm not the original author of this code, so I'm not sure what the purpose of everything in here is.